configure.py

```diff
@@ -57,550 +57,553 @@
     action='store_true',
     dest='with_ltcg',
     help='Use Link Time Code Generation. This feature is only available on Windows.')
 
-parser.add_option('--without-node-snapshot',
+parser.add_argument('--without-node-snapshot',
     action='store_true',
     dest='without_node_snapshot',
     help='Turn off V8 snapshot integration. Currently experimental.')
 
-intl_optgroup.add_option('--download',
+intl_group.add_argument('--download',
     action='store',
     dest='download_list',
     help=nodedownload.help())
 
-intl_optgroup.add_option('--download-path',
+intl_group.add_argument('--download-path',
     action='store',
     dest='download_path',
     default='deps',
-    help='Download directory [default: %default]')
+    help='Download directory [default: %(default)s]')
 
-parser.add_option_group(intl_optgroup)
+parser.add_argument_group(intl_group)
 
-parser.add_option('--debug-lib',
+parser.add_argument('--debug-lib',
     action='store_true',
     dest='node_debug_lib',
     help='build lib with DCHECK macros')
 
-http2_optgroup.add_option('--debug-nghttp2',
+http2_group.add_argument('--debug-nghttp2',
     action='store_true',
     dest='debug_nghttp2',
     help='build nghttp2 with DEBUGBUILD (default is false)')
 
-parser.add_option_group(http2_optgroup)
+parser.add_argument_group(http2_group)
 
-parser.add_option('--without-dtrace',
+parser.add_argument('--without-dtrace',
     action='store_true',
     dest='without_dtrace',
     help='build without DTrace')
 
-parser.add_option('--without-etw',
+parser.add_argument('--without-etw',
     action='store_true',
     dest='without_etw',
     help='build without ETW')
 
-parser.add_option('--without-npm',
+parser.add_argument('--without-npm',
     action='store_true',
     dest='without_npm',
     help='do not install the bundled npm (package manager)')
 
-parser.add_option('--without-report',
+parser.add_argument('--without-report',
     action='store_true',
     dest='without_report',
     help='build without report')
 
 # Dummy option for backwards compatibility
-parser.add_option('--with-snapshot',
+parser.add_argument('--with-snapshot',
     action='store_true',
     dest='unused_with_snapshot',
-    help=optparse.SUPPRESS_HELP)
+    help=argparse.SUPPRESS)
 
-parser.add_option('--without-snapshot',
+parser.add_argument('--without-snapshot',
     action='store_true',
     dest='without_snapshot',
-    help=optparse.SUPPRESS_HELP)
+    help=argparse.SUPPRESS)
 
-parser.add_option('--without-siphash',
+parser.add_argument('--without-siphash',
     action='store_true',
     dest='without_siphash',
-    help=optparse.SUPPRESS_HELP)
+    help=argparse.SUPPRESS)
 
 # End dummy list.
 
-parser.add_option('--without-ssl',
+parser.add_argument('--without-ssl',
     action='store_true',
     dest='without_ssl',
     help='build without SSL (disables crypto, https, inspector, etc.)')
 
-parser.add_option('--without-node-options',
+parser.add_argument('--without-node-args',
```

Comment:
This should remain `--without-node-options` as it is referring to the `NODE_OPTIONS` environment variable support.